cmake/Modules/LibAddPlugin.cmake

```diff
@@ -351,8 +351,6 @@ function (add_plugin PLUGIN_SHORT_NAME)
 	add_library (${PLUGIN_OBJS} OBJECT ${ARG_SOURCES})
 	add_dependencies (${PLUGIN_OBJS} kdberrors_generated)
 
-	add_dependencies(${PLUGIN_OBJS} readme_${PLUGIN_SHORT_NAME}.c)
```

Comment:
> Are you sure this dep is not needed or did you remove it because it did not work?

I deleted this line since it triggered warnings after I removed the setting for policy `CMP0046`.

> I think the idea was to retrigger plugin compilation after `README.md` is changed.

As far as I can tell, rebuilding works fine even after I delete the dependency. I tried the following:

1. Remove one of the commas in the example of the YAML ReadMe
2. Invoke the build system
3. The Markdown Shell Recorder correctly reports the syntax problem.

Can you think of an example where the rebuilding process does not work?